backend/services/interfaces/testSessionService.ts

```diff
@@ -60,4 +60,8 @@ export interface ITestSessionService {
   createTestSession(
     testSession: TestSessionRequestDTO,
   ): Promise<TestSessionResponseDTO>;
+
+  getTestSessionById(
```

Comment:
Add comments to document `getTestSessionById` - you can follow the format of the other function